docs/resources/telemetry_information.md

```diff
@@ -0,0 +1,77 @@
+# OpenVINO™ Telemetry {#openvino_docs_telemetry_information}
+
+@sphinxdirective
+
+To facilitate debugging and further development, OpenVINO™ asks its users for 
+a permission to collect telemetry data. It will not be collected 
+without an explicit consent on your part and will cover only OpenVINO™ usage information.
+It does not extend to any other Intel software, hardware, website usage, or other products. 
+
+Google Analytics is used for telemetry purposes. The data is collected anonymously 
+and does not allow identifying users it comes from. 
+
+Enable or disable Telemetry reporting
+======================================
+
+First-run consent
+--------------------------------------
+
+On the first run of an application that collects telemetry data, you will be prompted 
+to opt in or out of telemetry collection with the following telemetry message: 
+
+.. code-block:: console
+
+   Intel would like your permission to collect software performance and usage data
+   for the purpose of improving Intel products and services. This data will be collected
+   directly by Intel or through the use of Google Analytics. This data will be stored 
+   in countries where Intel or Google operate.
+
+   You can opt-out at any time in the future by running 'opt_in_out --opt_in'.
+   
+   More Information is available at docs.openvino.ai.
```

Comment:
@popovaan could we change the message in the telemetry library to this telemetry documentation page? Then it can be changed here too.

Reply:
That is a nice idea. We just need to remember to refer to specific documentation versions, not the "latest." 
With every release we need to change the link to the proper docs page.

Reply:
Yes, we can change the link every release both in telemetry library and in docs. Which link exactly should it be?

Reply:
As it involves hardcoding the final page address, I will merge this PR and publish already. If we decide to include it in the opt-in message, we can quickly update the article too.